src/BlockTypes/ProductRating.php

```diff
@@ -49,6 +49,15 @@ protected function get_block_type_supports() {
 				'background' => false,
 				'link'       => false,
 			),
+			'typography'             =>
+			array(
+				'fontSize'                        => true,
+				'__experimentalSkipSerialization' => true,
+			),
+			'spacing'                =>
+			array(
+				'margin' => true,
+			),
```

Comment:
`'__experimentalSkipSerialization' => true,` missing in `spacing` - let's also add it to `color` as mentioned above